src/csharp/Grpc.Tools/Grpc.Tools.csproj

```diff
@@ -0,0 +1,95 @@
+<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="..\Grpc.Core\Version.csproj.include" />
```

Comment:
please add 
```
<Import Project="..\Grpc.Core\Version.csproj.include" />
<Import Project="..\Grpc.Core\Common.csproj.include" />
```
- the Common.csproj.include should fix the linux build problem
- we want all csproj files to be consistent.

Reply:
`Common.csproj` enforces strong naming. If it's possible to avoid dealing with the SN in the context of MSBuild, I would. The test assembly can be strong-named, that's not a problem (although I am not really aware of any reason to SN an executable).

Reply:
Ah yes, the mono build fails w/o the framework path override trick in Common.csproj.include, so it is required. Also, essentially same hack is needed to build packages on a machine with the .NET but without the .NET SDK installed (i. e., your Windows test workers).